kncaputo/fitlit-refactor-tractor
# Calendar Form Event Handling

## What's this PR do?
- Allows users to pick a date from the calendar and updates user information

## Where should the reviewer start?
- `scripts.js`

## How should this be manually tested?
- Run `npm start` and choose a date from the calendar, ensure that the information has been updated

## Screenshots/GIFs
Carrie's dashboard view on 06/16/2019
<img width="1198" alt="Screen Shot 2020-10-30 at 8 19 25 PM" src="https://user-images.githubusercontent.com/58453566/97769060-5c1e6800-1aed-11eb-972b-04ada99d72c6.png">

Carrie's dashbaord view on 9/22/2019
<img width="1208" alt="Screen Shot 2020-10-30 at 8 19 38 PM" src="https://user-images.githubusercontent.com/58453566/97769065-67719380-1aed-11eb-9b24-8e3e8cbe2680.png">
